qualtran/bloqs/mcmt/multi_control_pauli.py

```diff
@@ -181,15 +194,15 @@ class MultiControlX(MultiControlPauli):
         target: single qubit target register.
     """
 
-    target_bloq: Bloq = field(init=False)
-
-    @target_bloq.default
-    def _X(self):
-        return XGate()
+    cvs: Union[HasLength, Tuple[int, ...]] = field(converter=_to_tuple_or_has_length)
 
     def __attrs_post_init__(self):
         pass
 
+    @property
```

Comment:
Should this be `@cached_property`? Or maybe even a class-level instance?

Reply:
I wouldn't use that microoptimization as motivation